src/main/resources/db/migration/0250/V265__SpeciesGrowthForm.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE public.species_growth_forms (
+    species_id INTEGER REFERENCES public.species ON DELETE CASCADE,
```

Comment:
This field does not seem to be getting the ID wrapped correctly within jOOQ generated classes.

Reply:
Try `BIGINT` instead of `INTEGER`.